ui/webui/test/check-progress

```diff
@@ -34,18 +34,19 @@ class TestInstallationProgress(anacondalib.VirtInstallMachineCase):
 
         i.open()
 
-        for page in i.steps[:-1]:
+        current_page = i.steps.WELCOME
```

Comment:
Can't this be current_page = get_current_page() , that seems cleaner and less hardcoding.